src/LayerEditor/ui/data/regions.py

```diff
@@ -0,0 +1,75 @@
+from LayerEditor.ui.data.region import Region
+
+
+class Regions:
+    def __init__(self, diagram):
+        self.regions = {Region.none.id: Region.none}
+        self._diagram = diagram
```

Comment:
Is the reference to the whole diagram necessary? Possibly keep just reference to the decomposition.

Reply:
Deleted in branch TB_LE_inicialization_and_IO